README.md

```diff
@@ -25,22 +25,34 @@ Psammead packages are split into:
 git clone git@github.com:bbc/psammead.git
 ```
 
-### :hammer: Setup Local Environment
+### Check npm and nvm versions
```

Comment:
We actually have a script to check this for you :) Perhaps we could make `sh scripts/enforceVersions.sh` a preinstall step? :)

Reply:
https://github.com/bbc/psammead/blob/0ac0405fb22068dc2e05cf56636af82257a99772/package.json#L15 added :) thank you!